From 614d897209c8d9d8b38bc583d0ce056a5d3445b6 Mon Sep 17 00:00:00 2001
From: Jan Friesse <jfriesse@redhat.com>
Date: Tue, 16 Apr 2013 16:08:24 +0200
Subject: [PATCH] Improve corosync-notifyd example

Example now contains default option -d, so corosync-notifyd init script
don't fall. Also description is improved a little bit.

Signed-off-by: Jan Friesse <jfriesse@redhat.com>
Reviewed-by: Fabio M. Di Nitto <fdinitto@redhat.com>
---
 tools/corosync-notifyd.sysconfig.example |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/tools/corosync-notifyd.sysconfig.example b/tools/corosync-notifyd.sysconfig.example
index 5cd7a2e..908bc8c 100644
--- a/tools/corosync-notifyd.sysconfig.example
+++ b/tools/corosync-notifyd.sysconfig.example
@@ -1,9 +1,10 @@
 #
-# See "man corosync-notifyd" for descriptions of
+# See "man corosync-notifyd" for detailed descriptions of
 # the options below.
 #
 # OPTIONS="-d -s -l -m <snmp manager address>"
 #
 
-OPTIONS=""
+# Send DBUS signals on all events (for SNMP traps, use -s)
+OPTIONS="-d"
 
-- 
1.7.1

